modules/nw-ptp-operator-metrics-reference.adoc

```diff
@@ -99,4 +99,10 @@ Possible values are 0 (`UNAVAILABLE`) and 1 (`AVAILABLE`).
 GNSS provides satellite-based positioning, navigation, and timing services globally.
 Possible values are 0 (`NOFIX`), 1 (`DEAD RECKONING ONLY`), 2 (`2D-FIX`), 3 (`3D-FIX`), 4 (`GPS+DEAD RECKONING FIX`), 5, (`TIME ONLY FIX`).
 |`{from="gnss",iface="ens2fx",node="compute-1.example.com",process="gnss"} 3`
+
+|`openshift_ptp_ha_profile_status`
+|Returns the current status of the highly available PTP Boundary Clock (T-BC) for each connected NIC.
```

Comment:
PTP-HA is actually  High Availability of the system clock when there are multiple time sources on different NICs.  YOu might to reword this .